third_party/terraform/resources/resource_dataflow_job.go

```diff
@@ -96,6 +96,18 @@ func resourceDataflowJob() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+
+			"network": {
```

Comment:
Make sure to update the documentation for the resource as well! https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/website/docs/r/dataflow_job.html.markdown